authentication/esignet-integration-impl/pom.xml

```diff
@@ -1,77 +0,0 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
is the eSignet integration impl moved to other repo? If yes, please added comments in which repo it has moved to.

Reply:
yes taheer, it's moved to https://github.com/mosip/esignet-plugins/tree/develop/mosip-identity-plugin confirmed by @balaji-alluru